sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -994,3 +995,121 @@ mod test_topology_manager_policy {
         }
     }
 }
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+/// KubernetesClusterDNSIp represents the --cluster-dns settings for kubelet.
+///
+/// The default is determined via information from IMDS in AWS.
```

Comment:
Do we need to mention where the default comes from? Feels like a different concern.